reporter/src/main/kotlin/StatisticsCalculator.kt

```diff
@@ -95,9 +82,8 @@ object StatisticsCalculator {
         )
     }
 
-    private fun getOpenVulnerabilities(ortResult: OrtResult, resolutionProvider: ResolutionProvider): Int =
-        ortResult.getVulnerabilities(omitExcluded = true).values.flatten()
-            .filterNot { resolutionProvider.isResolved(it) }.size
+    private fun getOpenVulnerabilities(ortResult: OrtResult): Int =
+        ortResult.getVulnerabilities(omitExcluded = true).values.flatten().size
```

Comment:
This needs to also pass `omitResolved = true` now in exchange for dropping `.filterNot { resolutionProvider.isResolved(it) }`.